prov/efa/src/rdm/efa_rdm_protocol.h

```diff
@@ -372,8 +342,10 @@ static_assert(sizeof(struct efa_rdm_handshake_hdr) == 8, "efa_rdm_handshake_hdr
 #endif
 
 struct efa_rdm_handshake_opt_connid_hdr {
-	uint32_t connid;
-	uint32_t padding; /* padding to 8 bytes boundary */
+	union {
+		uint32_t connid;
+		uint64_t padding;
```

Comment:
Is this change necessary? Your updated protocol doc still makes padding a 4 byte

Reply:
Right, it's a minor stylistic change in this case. I feel it's a more accurate representation of a primitive value aligned to an 8-byte boundary. Whereas the original syntax might be misconstrued as if there needs to be 4 bytes of padding/reserved space, regardless of the size of its preceding field(s).